scripts/badger/voter_chores.py

```diff
@@ -0,0 +1,52 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+
+def main():
+    voter = GreatApeSafe(r.badger_wallets.treasury_voter_multisig)
+    destination = GreatApeSafe(r.badger_wallets.treasury_ops_multisig)
+    voter.init_badger()
+
+    vlAURA = voter.contract(r.aura.vlAURA)
+    AURABAL = voter.contract(r.treasury_tokens.AURABAL)
+    GRAVI = voter.contract(r.sett_vaults.graviAURA)
+    AURA = voter.contract(r.treasury_tokens.AURA)
+
+    data = voter.badger.get_hh_data()
+    rewards = [x["token"] for x in data]
+
+    voter.take_snapshot(tokens=rewards)
+    destination.take_snapshot(tokens=rewards)
+
+    voter.badger.claim_bribes_hidden_hands(claim_from_strat=False)
```

Comment:
this reverts on `VirtualMachineError: revert: Invalid _claims`. maybe add a check here as well?

Reply:
include the check to see if there is indeed some data to claim for in 5d1f7e4e50ba4dbb2ef9b88b8e7e945d690f88a1